src/lib/create-config.ts

```diff
@@ -0,0 +1,197 @@
+import { parse } from 'pg-connection-string';
+import merge from 'deepmerge';
+import {
+    IUnleashOptions,
+    IUnleashConfig,
+    IDBOption,
+    ISessionOption,
+    IServerOption,
+    IVersionOption,
+    IAuthOption,
+    AuthType,
+    IImportOption,
+    IEmailOption,
+    ListeningPipe,
```

Comment:
I thought these might not be interfaces because they diverted from the convention. Perhaps rename to IListeningPipe and IListeningHost?

Reply:
Agreed